src/IssueViz.Security/OpenInIDE/Api/IOpenInIDEStateValidator.cs

```diff
@@ -20,16 +20,12 @@
 
 using System;
 using System.ComponentModel.Composition;
-using System.Diagnostics;
-using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.Binding;
-using SonarLint.VisualStudio.Core.InfoBar;
 using SonarLint.VisualStudio.Integration;
-using SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Api
 {
-    internal interface IOpenInIDEStateValidator : IDisposable
+    internal interface IOpenInIDEStateValidator
```

Comment:
The changes to the validator class and tests are straightforward - I just removed anything relating to the infobar and output window service.